notifications/endpoints/post-endpoint.ts

```diff
@@ -0,0 +1,43 @@
+import { NotificationInterface } from '../interfaces/notification.interface';
+import { PostNotificationPayloadInterface } from '../interfaces/post-notification-payload.interface';
+import { CommentNotificationPayloadInterface } from '../interfaces/comment-notification-payload.interface';
+import { ReplayNotificationPayloadInterface } from '../interfaces/replay-notification-payload.interface';
+import { CommentLikeNotificationPayloadInterface } from '../interfaces/comment-like-notification-payload.interface';
+
+export class PostEndpoint {
```

Comment:
duplicate class name can cause confusion. you can change the class name like post-module-notifications and post-module-errors